### PR TITLE
Update Cassandra image to include LDAP and Kerberos plugins.

### PR DESCRIPTION
**Pull request by zegelin**
Also enables repository-less Docker builds (i.e., on a local machine)

### CODE REVIEW

File: docker/cassandra/install-cassandra [hunk 1]
```diff
@@ -42,6 +62,40 @@ mkdir /etc/cassandra/logback.xml.d
...
+
+    # TODO: switch remaining Maven Central URLs to GitHub.
+    case ${cassandra_version} in
+    3.11.*)
+        curl -SLO "https://github.com/instaclustr/cassandra-ldap/releases/download/3.11.4.4/cassandra-ldap-3.11.4.jar"
+        ;;
+    3.0.*)
+        curl -SLO "https://search.maven.org/remotecontent?filepath=com/instaclustr/cassandra-ldap/3.0.18.0/cassandra-ldap-3.0.18.0.jar"
+        ;;
+    2.2.*)
```
Comment (benbromhead) [inline docker/cassandra/install-cassandra]: Is it worth doing 2.2? Given the rest of the operator doesnt support it?

File: docker/cassandra/install-cassandra [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+echoerr() { >&2 echo $@; }
+
 cassandra_version=$1
 
+case $cassandra_version in
+3.*)
+    # supported
+    ;;
+
+2.*)
```
Comment (benbromhead) [inline docker/cassandra/install-cassandra]: we should probably just remove this one

File: docker/cassandra/install-cassandra (hunk 1 above)
Comment (zegelin) [reply docker/cassandra/install-cassandra]: Added this before I tested 2.2. Then I realized that does some weird things on install and even fails to start, hence the test and comments above (line 12+).

Happy to remove this. Your call.

File: docker/cassandra/install-cassandra (hunk 2 above)
Comment (zegelin) [reply docker/cassandra/install-cassandra]: Put this here as more of a "here be dragons" to me or anyone else in the future.

**Pull request merged**